source/btle/btle.cpp

```diff
@@ -191,8 +191,11 @@ static void btle_handler(ble_evt_t *p_ble_evt)
 
             // Close all pending discoveries for this connection
             nRF5xGattClient& gattClient = ble.getGattClient();
+#if  defined(TARGET_MCU_NRF51_16K_S110) || defined(TARGET_MCU_NRF51_32K_S110)
```

Comment:
Thank you i was not certain of a pattern that would suit both gcc and armcc
compilers.


Il giorno ven 22 gen 2016 alle 12:46 Andres Amaya Garcia <
notifications@github.com> ha scritto:

> In source/btle/btle.cpp
> <https://github.com/ARMmbed/ble-nrf51822/pull/109#discussion_r50528604>:
>
> > @@ -191,8 +191,11 @@ static void btle_handler(ble_evt_t *p_ble_evt)
> >
> >              // Close all pending discoveries for this connection
> >              nRF5xGattClient& gattClient = ble.getGattClient();
> > +#if  defined(TARGET_MCU_NRF51_16K_S110) || defined(TARGET_MCU_NRF51_32K_S110)
>
> Perhaps condition could follow the same pattern of the others i.e.:
>
> #if !defined(TARGET_MCU_NRF51_16K_S110) && !defined(TARGET_MCU_NRF51_32K_S110)
>     /////// CODE HERE
> #endif
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ARMmbed/ble-nrf51822/pull/109/files#r50528604>.
>